.gitignore

```diff
@@ -1,2 +1,3 @@
 .gradle
 build
+.idea
```

Comment:
Note: should be in the global gitignore on the developer's computer. But let's keep that

Reply:
Good point, but I'd rather have such things within the local `.gitignore` since different teams have different setups.